tcms/static/js/utils.js

```diff
@@ -163,3 +163,34 @@ function treeViewBind() {
       }
     });
 }
+
+function filterTestCasesByProperty(testCases, property, value) {
+    const filterValue = value;
+    const filterBy = property;
+
+    for (let id in testCases) {
+        if (!testCases.hasOwnProperty(id)) {
```

Comment:
Maybe if we introduce some inheritance, or different object is passed to the function, the check will prevent looping unexpected items. I think its part of the eslint also.

Reply:
I don't think it's possible for this to return false, ever. As I said, you are looping over `testCases`, so whatever `id` is, will always be present in `testCases` and `hasOwnProperty` will be `true`. This seems like a check that protects us from something, but we don't know that from, and I would even argue that is not protecting us, because as I said, this will be always `true`.